packages/pages/src/build/build.ts

```diff
@@ -1,20 +1,26 @@
-import { Command } from "commander";
+import { CommandModule } from "yargs";
```

Comment:
Commander is what we changed to from yargs. I think you'll want to redo your merge. Looks like you reverted a lot of things.